util.c

```diff
@@ -109,8 +109,8 @@ usage(int code)
             " -z BYTES set the maximum job size in bytes (default is %d, max allowed is %d)\n"
             " -s BYTES set the size of each write-ahead log file (default is %d)\n"
             "            (will be rounded up to a multiple of 4096 bytes)\n"
-            " -c       compact the binlog (default)\n"
-            " -n       do not compact the binlog\n"
+            " -c       (DEPRECATED) compact the binlog (default)\n"
+            " -n       (DEPRECATED) do not compact the binlog\n"
```

Comment:
The "deprecated" term means "still works but not recommended", but the behavior was actually changed, isn't?

Reply:
Oh right, bad choice of words. In fact it was completely removed, not only discouraged.